bin/is-website-vulnerable.js

```diff
@@ -5,65 +5,70 @@
 const os = require('os')
 const debug = require('debug')('is-website-vulnerable')
 const argv = require('yargs').argv
+const inquirer = require('inquirer')
 const { Audit, RenderConsole, RenderJson, Utils } = require('../index')
 
-let url = process.argv[2]
-debug(`received url argument: ${url}`)
+const main = async () => {
```

Comment:
Note: the reason I did this is because using `inquirer` requires us to use promises, and since a large portion of the code is already asynchronous, I wrapped everything in an async function to make using `await` possible. I can change this, though, in case that's preferable.